roles/host-provision/templates/hostprovision_ansible_user.j2

```diff
@@ -0,0 +1 @@
+{{ hostprovision_ansible_user }}      ALL=(ALL)      NOPASSWD: ALL
```

Comment:
Any chance we could get a quick comment in this file to note that it's for bms-toolkit runs, and is necessary for initial provisioning and patching only?

Reply:
Done